cdk_stacks/cdk.context.json

```diff
@@ -2,5 +2,14 @@
   "vpc_name": "default",
   "opensearch_domain_name": "rag-vectordb",
   "jumpstart_llm_model_id": "META_TEXTGENERATION_LLAMA_2_7B_2_1_0",
-  "llm_endpoint_name": "llama2-7b"
+  "llm_endpoint_name": "llama2-7b",
+  "availability-zones:account=132415574961:region=us-east-1": [
```

Comment:
AZ info in the cdk.context.json should be removed.
It is automatically filled when synthesizing or deploying CDK Stacks.
Also, secret information should be exposed in the github.